custom/icds_reports/management/commands/backfill_supervisor_id.py

```diff
@@ -0,0 +1,66 @@
+# check all tables have non-empty awc_id
```

Comment:
Doesn't this need the function `Command()` somewhere to work as a management command?

Reply:
This was just for review. We planned to run these manually on a psql shell